extensions/data-workspace/src/dialogs/dialogBase.ts

```diff
@@ -83,31 +86,57 @@ export abstract class DialogBase {
 	 * created if no workspace is currently open
 	 * @param view
 	 */
-	protected createWorkspaceContainer(view: azdata.ModelView): azdata.FormComponent {
+	protected createWorkspaceContainer(view: azdata.ModelView): void {
 		const workspaceDescription = view.modelBuilder.text().withProperties<azdata.TextComponentProperties>({
 			value: vscode.workspace.workspaceFile ? constants.AddProjectToCurrentWorkspace : constants.NewWorkspaceWillBeCreated,
-			CSSStyles: { 'margin-top': '3px', 'margin-bottom': '10px' }
+			CSSStyles: { 'margin-top': '3px', 'margin-bottom': '0px' }
 		}).component();
 
 		this.workspaceInputBox = view.modelBuilder.inputBox().withProperties<azdata.InputBoxProperties>({
 			ariaLabel: constants.WorkspaceLocationTitle,
 			width: constants.DefaultInputWidth,
-			enabled: false,
+			enabled: !vscode.workspace.workspaceFile?.fsPath, // want it editable if no workspace is open
 			value: vscode.workspace.workspaceFile?.fsPath ?? '',
 			title: vscode.workspace.workspaceFile?.fsPath ?? '' // hovertext for if file path is too long to be seen in textbox
 		}).component();
 
-		const container = view.modelBuilder.flexContainer()
-			.withItems([workspaceDescription, this.workspaceInputBox])
-			.withLayout({ flexFlow: 'column' })
-			.component();
+		const browseFolderButton = view.modelBuilder.button().withProperties<azdata.ButtonProperties>({
```

Comment:
to be accurate this is actually browseFileButton

Reply:
after discussing with Raden, we decided to make this a browse folder button so the user can only pick the folder to create the new workspace file in. This will make it less likely that they'll choose an existing workspace